### PR TITLE
owner: Update the interval of printing log

### PR DESCRIPTION
**Pull request by zimulala**

### CODE REVIEW

File: owner/manager.go [hunk 1]
```diff
@@ -130,8 +134,11 @@ func NewSession(ctx goctx.Context, logPrefix string, etcdCli *clientv3.Client, r
 		if err == nil {
 			break
 		}
-		log.Warnf("%s failed to new session, err %v", logPrefix, err)
+		if failedCnt%intervalCnt == 0 {
+			log.Warnf("%s failed to new session, err %v", logPrefix, err)
+		}
 		time.Sleep(200 * time.Millisecond)
```
Comment (shenli) [inline owner/manager.go]: Is this the same with newSessionRetryInterval?

File: owner/manager.go [hunk 2]
```diff
@@ -120,6 +122,8 @@ func setManagerSessionTTL() error {
 func NewSession(ctx goctx.Context, logPrefix string, etcdCli *clientv3.Client, retryCnt, ttl int) (*concurrency.Session, error) {
 	var err error
 	var etcdSession *concurrency.Session
+	failedCnt := 0
+	intervalCnt := int(3 * time.Second / newSessionRetryInterval)
```
Comment (coocood) [inline owner/manager.go]: Why not define `intervalCnt` as constant as well?
And I think `logInterval` maybe a better name.

**Comment by zimulala** [comment]
PTAL @coocood

**Comment by coocood** [comment]
LGTM

**Pull request merged**